astarte-utils/AstarteGenericProducer.cpp

```diff
@@ -41,12 +41,25 @@ QHash<QByteArray, QVariant::Type> AstarteGenericProducer::mappingToType() const
     return m_mappingToType;
 }
 
-bool AstarteGenericProducer::sendData(const QVariant &value, const QByteArray &target,
-        const QDateTime &timestamp, const QVariantHash &metadata)
+QHash<QByteArray, QVariant::Type> AstarteGenericProducer::mappingToArrayType() const
+{
+    return m_mappingToArrayType;
+}
+
+bool validateTarget(const QByteArray &target)
```

Comment:
this function should be either static (with C meaning), or private and static (C++) meaning.

Reply:
this is a predicate, it should be called isValidTarget